Content.Server/Warps/WarpPointComponent.cs

```diff
@@ -10,9 +10,16 @@ public sealed partial class WarpPointComponent : Component
         public string? Location;
 
         /// <summary>
-        ///     If true, ghosts warping to this entity will begin following it.
+        /// If true, ghosts warping to this entity will begin following it.
         /// </summary>
         [DataField]
         public bool Follow;
+
+        /// <summary>
+        /// Should this warp point be accessable to ghosts only?
+        /// Useful where you want things like a ghost to reach only like CentComm
+        /// </summary>
+        [DataField]
+        public bool GhostOnly;
```

Comment:
i dont know if i agree with sloth that this should be a whitelist but maybe a more descriptive name? The actual difference isn't that these things are "ghost only" but rather that they are non-fixed locations or tied to objects.

Reply:
I figure a blacklist would be better.
In more cases than not you want to opt into as many warp points as possible and exclude others.
What isn't done here yet is changing maps like centcomm that have warp points that are fixed locations.